src/test/javascript/portal/filter/DateFilterSpec.js

```diff
@@ -55,11 +55,6 @@ describe("Portal.filter.DateFilter", function() {
             expect(filter.getCql()).toBe("column_name >= '1999-01-01T00:00:00Z'");
         });
 
-        it('gives data layer CQL', function() {
```

Comment:
I set out to leave all the same existing tests in `WmsDateFilter` but could not work out a way to get the resulting call to `getEndOfToDate` to work